srcpkgs/sayonara-player/patches/00-musl-execinfo.patch

```diff
@@ -0,0 +1,33 @@
+--- src/CMakeLists.txt
```

Comment:
This looks upstreamable, have you PR'd it? :)

Reply:
Not yet, I've just created the package today, but will try to get it upstreamed for sure.

Reply:
@ericonr https://gitlab.com/luciocarreras/sayonara-player/-/merge_requests/160

Reply:
@ericonr the author have released a new version: `1.7.0stable2` with upstreamed patch, I've updated this PR accordingly.